src/MongoDB/BulkWrite.c

```diff
@@ -451,26 +454,21 @@ static phongo_create_object_retval php_phongo_bulkwrite_create_object(zend_class
 #else
 	{
 		zend_object_value retval;
-		retval.handle = zend_objects_store_put(intern, (zend_objects_store_dtor_t) zend_objects_destroy_object, php_phongo_bulkwrite_free_object, NULL TSRMLS_CC);
+		retval.handle   = zend_objects_store_put(intern, (zend_objects_store_dtor_t) zend_objects_destroy_object, php_phongo_bulkwrite_free_object, NULL TSRMLS_CC);
 		retval.handlers = &php_phongo_handler_bulkwrite;
 
 		return retval;
 	}
 #endif
 } /* }}} */
 
-static HashTable *php_phongo_bulkwrite_get_debug_info(zval *object, int *is_temp TSRMLS_DC) /* {{{ */
+static HashTable* php_phongo_bulkwrite_get_debug_info(zval* object, int* is_temp TSRMLS_DC) /* {{{ */
 {
-#if PHP_VERSION_ID >= 70000
-	zval                      retval;
-#else
-	zval                      retval = zval_used_for_init;
-#endif
-	php_phongo_bulkwrite_t   *intern = NULL;
-
+	zval                    retval = ZVAL_STATIC_INIT;
```

Comment:
I suppose this is a bit of a WTF that we don't use `ZVAL_RETVAL_TYPE` here, but that can always be helped later by renaming `retval` to something else. This isn't a `retval` in the normal sense of a `PHP_METHOD`.

Nothing to change here, just making a note.